doc/api/process.md

```diff
@@ -1647,21 +1647,22 @@ This feature is not available in [`Worker`][] threads.
 added: REPLACEME
 -->
 
-* `inc` {integer} The new nice value for the process.
+* `inc` {integer} The nice value which is added to the current nice value.
+
+Returns the new nice value of the process.
 
 The `process.nice()` method sets or gets the nice value of the process.
 (See nice(2).) The `inc` is supposed to be an integer, consisting of the
-_difference_ you want to add to the current nice value.
+_difference_ the user wants to add to the current nice value.
```

Comment:
maybe "to be added to the current nice value"?

Reply:
That's cool. Changed: 2a8e90ad0ab93f07b67628b0e7c876603da47c8a